.github/Makefile

```diff
@@ -6,19 +8,17 @@ SHELL = /bin/bash
 #############################
 PROJECT_MAJOR_VERSION ?= $(shell echo $(CURRENT_RELEASE) | cut -f1 -d.)
 PROJECT_MINOR_VERSION ?= $(shell echo $(CURRENT_RELEASE) | cut -f2 -d.)
-PROJECT_PATCH_VERSION ?= $(shell echo $(CURRENT_RELEASE) | cut -f3 -d.)
```

Comment:
minor release don't need the patch version since it's always 0 and unused in the script.